packages/fuel-gauge/src/predicate-conditional-inputs.test.ts

```diff
@@ -22,11 +22,8 @@ describe('PredicateConditionalInputs', () => {
 
     const amountToTransfer = 1000;
 
-    const predicate = new Predicate({
-      bytecode: PredicateConditionalInputsAbi__factory.bin,
-      abi: PredicateConditionalInputsAbi__factory.abi,
-      provider,
-      configurableConstants: { MAKER: aliceWallet.address.toB256() },
+    const predicate = PredicateConditionalInputsAbi__factory.createInstance(provider, undefined, {
+      MAKER: aliceWallet.address.toB256(),
     });
```

Comment:
Not a now issue - but what do you think of this DX?

Maybe an options argument might be nicer

```ts
PredicateConditionalInputsAbi__factory.createInstance(provider, {
  configurables: {
    MAKER: aliceWallet.address.toB256(),
  }
})
```

Reply:
I don't know, really. A GH discussion on this would be nice to flesh it out.

Reply:
https://github.com/FuelLabs/fuels-ts/discussions/2792